arangograph/cloud-migration-tool.md

```diff
@@ -245,6 +245,37 @@ error code `11` (ERROR_FORBIDDEN).
 The `--source.server-mode` option allows you to specify the desired server mode.
 Allowed values are `readonly` or `default`.
 
+### Supported environment variables
+
+The `arangosync-migration` tool supports the following environment variables:
+
+- `$OASIS_API_KEY`
```

Comment:
@nikita-vanyasin should we rename these these (i.e., non Oasis)?
We can keep supporting the old ones for backwards comp, but at least in the docs I would mention the new ones...

Reply:
@joerg84 sure! I'll create a ticket for migration tool.